src/frontend/utils/conversejs/converse.ts

```diff
@@ -0,0 +1,63 @@
+import 'converse.js/dist/converse.min.js';
+import { getDecodedJwt } from 'data/appData';
+import { XMPP } from 'types/XMPP';
+import { converse } from './../window';
+import { chatPlugin } from './converse-plugins/chatPlugin';
+import { logoutPlugin } from './converse-plugins/logoutPlugin';
+import { marshaJoinDiscussionPlugin } from './converse-plugins/marshaJoinDiscussionPlugin';
+
+export const converseMounter = () => {
+  let hasBeenInitialized = false;
+
+  return (xmpp: XMPP) => {
+    if (hasBeenInitialized) {
+      // tslint:disable-next-line: no-console
+      console.log('Chat has already been initialized.');
```

Comment:
If this is not needed anymore you should probably refactor this function an other way ?

Reply:
As mentioned before, i have changed this PR to make it able to stay in converse 7.0.6. 
This comment is therefore not applicable here anymore, but I keep it in mind for the second PR.